src/XGrid.cu

```diff
@@ -0,0 +1,298 @@
+/* 
+
+Implements the grid-Correlation onto a GPU using CUDA. 
```

Comment:
how is this different from GMul?

Reply:
We do not need GMul, it is no longer needed.